### PR TITLE
make response textarea a custom vue component

### PR DESCRIPTION
**Pull request by championswimmer**
This makes the templating of response easier

The `body` should contain content devoid of any names/addresses
This can be read off from a .inc/.md template

In the textarea contents, we can rearrange `name`, `address` and other details before and after the `body`

Signed-off-by: Arnav Gupta <arnav@codingblocks.com>

### CODE REVIEW

File: public/app.js [hunk 1]
```diff
@@ -19,6 +19,23 @@ var i18n = new VueI18n({
   fallbackLocale: 'en'
 });
 
+Vue.component('textarea-response', {
+  props: [
+    'address',
+    'body'
+  ],
+  template: `
```
Comment (championswimmer) [inline public/app.js]: It'll be hard to put newlines here without ES6 template strings, so using those here

Comment (championswimmer) [reply public/app.js]: newlines as in, after {{address}} and before {{body}}

Comment (captn3m0) [reply public/app.js]: @championswimmer Can you add the address in the service-template, and the bank name in the bank template:
``
MD and Chairman,
{{bankName}}
```

File: _includes/draft-notice-bank.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
-Chairman and Managing Director
+Chairman and Managing Director,
+((addressee))
 
 Dear Sir/Madam,
 
-Subject: Threats to make bank accounts inoperable without Aadhaar 
+Subject: Threats to make bank accounts inoperable without Aadhaar
```
Comment (championswimmer) [inline _includes/draft-notice-bank.md]: SOMEONE TELL ME WTF WAS WRONG EARLIER :( :( :( :(

Comment (championswimmer) [reply _includes/draft-notice-bank.md]: I SPENT 2 HOURS ON THIS ONE LINE OF **APPARENTLY NO CHANGE** wow

**Pull request merged**